libs/components/forms/src/lib/modules/input-box/input-box-host.service.ts

```diff
@@ -8,13 +8,17 @@ import { SkyInputBoxComponent } from './input-box.component';
  */
 @Injectable()
 export class SkyInputBoxHostService {
-  private host: SkyInputBoxComponent;
+  #host: SkyInputBoxComponent | undefined;
 
   public init(host: SkyInputBoxComponent): void {
-    this.host = host;
+    this.#host = host;
   }
 
   public populate(args: SkyInputBoxPopulateArgs): void {
-    this.host.populate(args);
+    if (!this.#host) {
+      throw new Error('Input box host has not been initialized');
```

Comment:
Let's make this error message more helpful. Something like, "Cannot populate the input box because `SkyInputBoxHostService` has not yet been initialized. Try running the `populate` method within an Angular lifecycle hook, such as `ngOnInit`."